packages/atomic/cypress/support/commands.ts

```diff
@@ -20,18 +20,18 @@ declare global {
 
 Cypress.Commands.add('expectSearchEvent', (actionCause) => {
   cy.wrap(AnalyticsTracker)
-    .invoke('getLastSearchEvent')
+    .invoke('getLastSearchEvent', actionCause)
```

Comment:
As stated [above](https://github.com/coveo/ui-kit/pull/1688/files#r789146098), I also decided to make AnalyticsTracker a bit more specific and take the `actionCause` or `eventType` as a parameter.

I could make it fully specific to even further reduce flakiness by making `getLastSearchEvent`, `getLastCustomEvent` and `getLastClickEvent` take an object as a parameter and assert on every key-value of the object. I decided not to do this since this is a bigger change and would require some more code to recursively compare objects, but I can do it if you prefer that.